admin-app/src/redux/store.ts

```diff
@@ -1,14 +1,14 @@
 import { configureStore } from "@reduxjs/toolkit";
 import chatReducer from "../shared/chat/redux/chat-page/chat-list.slice";
 import authReducer from "../shared/auth/redux/auth.slice";
-import activeChatsReducer from "../shared/chat/redux/active-chats.slice";
+import hiddenChatsReducer from "shared/chat/redux/hidden-chats.slice";
 import chatInfoReducer from "../shared/chat/redux/settings-page/chat.slice";
 
 export const store = configureStore({
 	reducer: {
 		chats: chatReducer,
 		auth: authReducer,
-		activeChats: activeChatsReducer,
+		hiddenChats: hiddenChatsReducer,
```

Comment:
i think chat and hiddenchat must be in one reducer
i think you need create filter function to chats array